test/examples/regression/issue262_static_method/issue262_static_method.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+import issue262_static_method_boost_python as boost
+import issue262_static_method_pybind11 as py11
+
+
+class TestIssue262(unittest.TestCase):
+
+    def _test_issue262(self, binding):
+        # Should be skipped as this static method overloads instance method
+        if binding is py11:
+            with self.assertRaises(TypeError):
+                binding.Integer.add(1, 2)
+        elif binding is boost:
+            # For some reason, Boost.Python allows instance and static method overloading
+            # For the details see: https://github.com/personalrobotics/chimera/issues/262
+            self.assertEqual(binding.Integer.add(1, 2), 3)
```

Comment:
@psigen Do you think we should disable instance and static method overloading for both bindings? For some reason, Boost.Python allows this, which I don't know why because it shouldn't be supported it's not valid Python syntax I believe.

Reply:
Never mind. Boost.Python also doesn't allow this case as well. Updated the templates and tests.